src/getCoinCombination.test.js

```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return [1, 0, 0, 0] for 1 cent',() => {
```

Comment:
it('returns correct combination for 1 cent',() => {

Reply:
Done